plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/cross-sells.tsx

```diff
@@ -21,7 +21,7 @@ const collection = {
 	),
 	keywords: [ 'boost', 'promotion' ],
 	scope: [],
```

Comment:
Good point. I thought about adding it but then I thought your PR might be merged first. But it makes sense to add in this PR as well, in case your PR isn't merged first. Also, It would be easier for reviewer to review this PR.